updating azure/aws preflight onboarding checks instructions (#2851)
EnterpriseDB/docs: product_docs/docs/biganimal/release/getting_started/preparing_cloud_account/01_preparing_azure/index.mdx

    ```
    The required arguments are:

    | Argument | Description |
    | -------- | ----------- |
    | &lt;target-subscription> | Azure subscription ID of your BigAnimal deployment. |
    | &lt;region> | Azure region where your clusters are being deployed. See [Supported regions](/biganimal/release/overview/03a_region_support) for a list of possible regions. |

    Possible options are:

    | Options | Description |
    | ------- | ----------- |
    | `-h` or `--help`| Displays the command help. |
    | `-i` or `--instance-type` | Azure VM instance type for the BigAnimal cluster. The `help` command provides a list of possible VM instance types. Choose the instance type that best suits your application and workload. Choose an instance type in the memory optimized ESv3 or ESv4 series for large data sets. Choose from the compute optimized FSv2 series for compute-bound applications. Choose from the general purpose DSv3 or DSv4 series if you don't require memory or compute optimization. See [Sizes for virtual machines in Azure](https://docs.microsoft.com/en-us/azure/virtual-machines/sizes) for information to help you choose the appropriate instance type. |

>     | `-a` or `--high-availability` | *DEPRECATED* - Enables high availability for the cluster. See [Supported cluster types](/biganimal/release/overview/02_high_availability) for more information.|

Reviewer: ```suggestion
    | `-a` or `--high-availability` | *DEPRECATED* - Enables high availability for the cluster. Replaced with `-x` or `--cluster-architecture` command.|
```

>     | `-x` or `--cluster-architecture` | Defines the Cluster architecture and can be [ single | ha | eha ]. See [Supported cluster types](/biganimal/release/overview/02_high_availability) for more information.|

Reviewer: ```suggestion
    | `-x` or `--cluster-architecture` | Defines the cluster architecture and can be [ single | ha | eha ]. See [Supported cluster types](/biganimal/release/overview/02_high_availability) for more information.|
```
